packages/vite-node/src/client.ts

```diff
@@ -83,7 +83,9 @@ export class ViteNodeRunner {
 
     // disambiguate the `<UNIT>:/` on windows: see nodejs/node#31710
     const url = pathToFileURL(fsPath).href
-    const exports: any = {}
+    const exports: any = {
```

Comment:
While we are at it. Maybe it should be `Object.create(null)`? 

If I run it in native Node, `module` has a null prototype

Reply:
Also, can you create a test case to cover this?